Fix misc issues (#124)
dortania/OpenCore-Install-Guide: troubleshooting/troubleshooting.md

* config.plist -> Kernel -> Quirks -> DisableRtcChecksum -> true

**Note**: If you still have issues, you'll need to use [RTCMemoryFixup](https://github.com/acidanthera/RTCMemoryFixup/releases) and exclude ranges.

For a more in-depth guide, see here:

* [Fixing RTC/CMOS Resets](https://dortania.github.io/OpenCore-Post-Install/misc/rtc.html)

## Synaptics PS2 based trackpad doesn't work

> You can try to use the [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl).

Reviewer: ```suggestion
You can try to use [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl).
```
Clarification on what exactly this does?

> First, you have to open DeviceManager, and head to the following:

Reviewer: ```suggestion
First, you have to open Device Manager, and head to the following:
```

```

> Device Manager -> Mice and other pointing devices -> Properties -> Details > BIOS device name

Reviewer: ```suggestion
Device Manager -> Mice and other pointing devices -> Double click on your trackpad -> Properties -> Details > BIOS device name
```

```

> Then grab the [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl)

Reviewer: ```suggestion
Then grab [SSDT-Enable_DynamicEWMode.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-Enable_DynamicEWMode.dsl)
```

By default, this uses PCI0.LPCB.PS2K for the pathing. you'll want to rename accordingly.

```
External (_SB_.PCI0.LPCB.PS2K, DeviceObj) <- Rename this

    Name(_SB.PCI0.LPCB.PS2K.RMCF, Package()  <- Rename this

```

> 

Reviewer: ```suggestion

Then compile with MaciASL, copy to your OC/ACPI folder, and add it to your config, and you should be good to go.

```

> * Note: Although this will work for some case, [the trackpad may be laggy and you can't use the physical buttons](https://github.com/acidanthera/bugtracker/issues/890). If you can't live without the trackpad, this may be better:

Reviewer: ```suggestion
* Note: Although this will work for most cases, the trackpad may be laggy and you may not be able to use the physical buttons ([more details](https://github.com/acidanthera/bugtracker/issues/890)). If you can live without the trackpad, this may be better:
```

Can or can't? From looking at the SSDT it appears to me to be can, but that makes no sense if you are trying to fix your touchpad...

Reviewer 2: Sorry for my typo, it must be can instead of can't and if you want to look at my pull request, it is [here](https://github.com/dortania/OpenCore-Install-Guide/pull/122)

> You have to find the path of your mouse just like the first solution. We then grab [SSDT-DisableTrackpadProbe.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-DisableTrackpadProbe.dsl). By default, this uses PCI0.LPCB.PS2K so you have to rename that to make sure that the SSDT work:

Reviewer: ```suggestion
Find the ACPI path of your mouse (see above), then grab [SSDT-DisableTrackpadProbe.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-DisableTrackpadProbe.dsl). By default, this uses PCI0.LPCB.PS2K so you have to change that to your ACPI path if necessary:
```

Again, what does this SSDT do?

```
External (_SB_.PCI0.LPCB.PS2K, DeviceObj) <- Rename this

    Name(_SB.PCI0.LPCB.PS2K.RMCF, Package() <- Rename this
```

> ## Fix for Dell breakless PS2 keyboard keys

Reviewer: What are `breakless PS2 keyboard keys`?

Author: When the key press doesn't release(ie. infinite press), clarified here:
* https://github.com/dortania/OpenCore-Install-Guide/pull/124/commits/8234374410bd67176961541418cf224f16bfb895

First of all, you need to find the path to your ACPI keyboard object in the Device Manager:

```

> Device Manager -> Keyboards -> Properties -> Details > BIOS device name

Reviewer: ```suggestion
Device Manager -> Keyboards -> Double click on your keyboard -> Properties -> Details > BIOS device name
```

```

> After this, you would like to grab the [SSDT-KEY-DELL-WN09.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-KEY-DELL-WN09.dsl) and fix some small things in the SSDT:

Reviewer: ```suggestion
After this, gran [SSDT-KEY-DELL-WN09.dsl](https://github.com/acidanthera/VoodooPS2/blob/master/Docs/ACPI/SSDT-KEY-DELL-WN09.dsl) and change the ACPI path to the one found above as needed:
```

![](../images/troubleshooting/troubleshooting-md/error.png)

## Selecting Startup Disk doesn't apply correctly

> If you're having issues with Start Up disk correctly applying your new boot entry, this is most likely caused by a missing DevicePathsSupported in your I/O Registry. To resolve this, ensure you are using `PlatformInfo -> Automatic -> True`

Reviewer: ```suggestion
If you're having issues with Startup Disk correctly applying your new boot entry, this is most likely caused by a missing `DevicePathsSupported` in your I/O Registry. To resolve this, ensure you are using `PlatformInfo -> Automatic -> True`
```

> Example of missing DevicePathsSupported:

Reviewer: ```suggestion
Example of missing `DevicePathsSupported`:
```
